package/patterns-yast.spec

```diff
@@ -136,6 +136,9 @@ Suggests:       snapper
 # FATE 304350
 Suggests:       sblim-sfcb
 Suggests:       cim-schema
+%ifarch s390x
```

Comment:
Thanks! But everywhere else we are using `%ifarch s390 s390x` Can you please change it for consistency?

Reply:
All the other patterns are using %ifarch s390x so I guess it depends on whether we want to be consistent with yast or with patterns.